### ISSUE TITLE
ISO Issue

### ISSUE THREAD

**Issue by zaposnik** [pepbioalerts/occTest#4]
Hello, I am working with occTest to implement a data cleaning workflow for occurrence data. I have 134 species and when I run the code 79 of them run fine, but for 55 of them, when the geographic outlier test is running I get the error:
 
geographic outliers detection started
Error with species: Thlypopsis_pectoralis - Cannot parse CN-TW into a known Country

I have checked the data frame and none of the ISO codes are CN-TW, as well as the data that is pulled in from:
"https://github.com/pepbioalerts/vignetteXTRA-occTest/raw/main/ext/ctry_nrec_info.rds" and there doesn't seem to be anything with CN-TW as a country code. I am wondering if anyone else has had this issue? Please let me know what other information would be useful,  don't post on github much so apologies if this could use more context.

Thanks!
Zach

**Comment by LinjieLi5718** [comment]
i have the same question but not solve yet.

**Comment by pepbioalerts** [comment]
This was because of some issues with iso_a2 naming conventions. It should be fixed now thanks to @mdpillet

**Comment by pepbioalerts** [comment]
Checked and it works now.
